converter_2.go

```
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+)
+
+func myStrToInt2(str string) (int, error) {
+	var b int
+	x, err := fmt.Sscanf(str, "%d", &b)
+	if str == "" {
+		return 1, error(nil)
```

The same question as above.

---

Please return an error, instead of special values